clients/admin-ui/src/features/config-wizard/AuthenticateOktaForm.tsx

```diff
@@ -0,0 +1,213 @@
+    }
+  };
+
+  return (
+    <Formik
+      initialValues={initialValues}
+      validationSchema={ValidationSchema}
+      onSubmit={handleSubmit}
+    >
+      {({ isValid, isSubmitting, dirty }) => (
+        <Form data-testid="authenticate-okta-form">
+          <Stack spacing={10}>
+            {isSubmitting ? (
+              <ScannerLoading
+                title="System scanning in progress"
+                onClose={handleCancel}
+              />
+            ) : null}
+
+            {scannerError ? <ScannerError error={scannerError} /> : null}
+            {!isSubmitting && !scannerError ? (
+              <>
+                <Heading size="lg">Authenticate Scanner</Heading>
+                <Accordion allowToggle border="transparent">
+                  <AccordionItem>
+                    {({ isExpanded }) => (
+                      <>
+                        <h2>
+                          The scanner can be connected to your sign-on provider
+                          to automatically scan and create a list of all systems
+                          that your team are using that may contain personal
+                          data.
+                          <AccordionButton
+                            display="inline"
+                            padding="0px"
+                            ml="5px"
+                            width="auto"
+                            color="complimentary.500"
+                          >
+                            {isExpanded ? `(show less)` : `(show more)`}
+                          </AccordionButton>
+                        </h2>
+                        <AccordionPanel padding="0px" mt="20px">
+                          Currently the scanner supports certain Okta. We will
+                          be adding support for other sign-on providers (e.g.
+                          Auth0,Microsoft, Google) shortly, please let us know
+                          if you have specific requests.
+                        </AccordionPanel>
+                      </>
+                    )}
+                  </AccordionItem>
+                </Accordion>
+                <Divider m="20px 0px !important" />
+                <Text m="0px !important">
+                  In order to run the scanner, please provide your Okta token.
+                  You can find{" "}
+                  <DocsLink href={DOCS_URL_OKTA_TOKEN}>
+                    instructions here{" "}
+                  </DocsLink>{" "}
+                  on how to retrieve a suitable token from your Okta
+                  administration
+                </Text>
+                panel.
+                <Stack>
+                  <CustomTextInput
+                    name="orgUrl"
+                    label="Org URL:"
+                    tooltip="Need text"
+                  />
+                  <CustomTextInput
+                    name="token"
+                    label="Okta token:"
```

Comment:
Should this field be masked?

Reply:
Yeah, strikes me as something that might be pasted in during a demo that shouldn't be seen.